config/config.go

```diff
@@ -0,0 +1,102 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/gol4ng/logger"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"go.etf1.tf1.fr/etf1-platform/pkg/config"
+)
+
+const AppName = "kafka-mongo-watcher"
+
+var AppVersion = "wip"
```

Comment:
i think app info are relative to a cmd not config

Reply:
Yep but it's simpler to use the constant name `config.AppName` from other packages while I cannot use `main.AppName` from other packages so I prefer to keep them here :-)